src/ui/Combobox/ComboboxInput.ts

```diff
@@ -81,15 +76,23 @@ export class ComboboxInput {
     this.combobox.onInputBlur();
   }
 
-  private handleKeyboardUpDownArrows(event: KeyboardEvent) {
+  private handleKeyboardDirection(event: KeyboardEvent) {
     switch (event.which) {
       case KEYBOARD.DOWN_ARROW:
         event.preventDefault();
-        this.combobox.values.moveActiveValueDown();
+        this.combobox.values.focusNextValue();
         break;
       case KEYBOARD.UP_ARROW:
         event.preventDefault();
-        this.combobox.values.moveActiveValueUp();
+        this.combobox.values.focusPreviousValue();
+        break;
+      case KEYBOARD.HOME:
+        event.preventDefault();
+        this.combobox.values.focusFirstValue();
+        break;
+      case KEYBOARD.END:
```

Comment:
When are the `HOME` and `END` keys used? Are these for accessibility?

Reply:
Yes, to get to the first and last results